go.mod

```diff
@@ -64,6 +64,6 @@ require (
 	gopkg.in/yaml.v3 v3.0.1 // indirect
 )
 
-// replace github.com/hyperledger/ursa-wrapper-go => github.com/ashcherbakov/ursa-wrapper-go v0.3.1
+replace github.com/hyperledger/ursa-wrapper-go => github.com/kgoncharov/ursa-wrapper-go v0.3.1
```

Comment:
will this be permanent?

Reply:
I'll delete it as soon as ursa-wrapper-go PR will be merged